WhaleSpotting/react-app/src/components/Navbar.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from "react";
+import { Link, useLocation } from "react-router-dom";
+import "../styles/Navbar.scss";
+
+export default function Navbar(): JSX.Element {
+    const location = useLocation();
+    const removeSlash = location.pathname.slice(1);
+    const pageName = removeSlash === "" ? "Home" : removeSlash.substr(0, 1).toUpperCase() + removeSlash.substr(1).toLowerCase();
+    const [currentPage, setCurrentPage] = useState(pageName);
+
+    function HandleLinkClick(currentPage: string) {
+
+        setCurrentPage(currentPage);
+        console.log(currentPage);
+    }
+    const loggedIn = false;
+
+    useEffect(() => {
+        setCurrentPage(pageName);
+    }, [location]);
+    return (
+        <div className="header">
+            <nav className="nav-bar">
+                <div className="group1">
+                    <Link to="./Home" data-testid="home"
+                        className={currentPage === "Home" ? "nav-link-selected" : "navbar-link"}
+                        onClick={() => HandleLinkClick("Home")}>Home</Link>
+                    <Link to="./Map"
```

Comment:
```suggestion
                    <Link to="/Map"
```
All the links don't need the `.` AFAICT